sdk_container/src/third_party/portage-stable/dev-lang/python/python-3.12.0.ebuild

```diff
@@ -28,7 +28,7 @@ S="${WORKDIR}/${MY_P}"
 
 LICENSE="PSF-2"
 SLOT="${PYVER}"
-KEYWORDS="~alpha ~amd64 arm arm64 ~hppa ~ia64 ~loong ~m68k ~mips ppc ~ppc64 ~riscv ~s390 sparc ~x86"
+KEYWORDS="~alpha ~amd64 arm arm64 ~hppa ~ia64 ~loong ~m68k ~mips ppc ppc64 ~riscv ~s390 sparc ~x86"
```

Comment:
It's interesting they stabilise python 3.12 for arm64 but not for amd64.

Reply:
There probably isn't one team that takes care of stabilization process for both amd64 and arm64. When updating packages I quite often see the package being stable for one arch, but not another. It usually straightens out within a week or two.

Also, it's good to note that 3.12 is becoming stable. It's something we could soon switch to.